.github/workflows/month_tag.yml

```
@@ -0,0 +1,25 @@
+on:
+  schedule:
+    - cron:  '0 0 1 * *'
+
+name: monthly-release
+jobs:
+  monthly-release:
+    name: monthly release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: format calver prefix
+        id: prefix
+        run: |
+          echo "::set-output name=date::$(date +'%Y.%-m')"
+      - name: extract tag message
+        id: tagmessage
+        run: |
+          echo "::set-output name=tagmessage::$(git log --pretty='format: - %s' --first-parent $(git describe --tags --abbrev=0)..HEAD)"
```

```suggestion
          echo "tagmessage=$(git log --pretty='format: - %s' --first-parent $(git describe --tags --abbrev=0)..HEAD)" >> $GITHUB_OUTPUT
```

`set-output` commands via stdout is deprecated and will start failing soon (https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) so use the new approach of appending to the [`$GITHUB_OUTPUT` environment file](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files).

c.f. https://github.com/scikit-hep/pyhf/pull/2155 for further examples.